### ISSUE TITLE
Entry options on mobile

### ISSUE THREAD

**Issue by angrybrad** [craftcms/cms#1124]
> Created by: Steve Adams (pixeljitsu@gmail.com) on 2015/08/05 16:40:23 +0000
> Votes at time of UserVoice import: 3

-----------------------------------

The entry options sidebar should be moved to an off canvas menu for mobile.  This would allow the entry options to be available at any point in the page scroll.

http://tympanus.net/Development/OffCanvasMenuEffects/sideslide.html
